src/status_im/ui/screens/db.cljs

```diff
@@ -87,6 +90,11 @@
 (spec/def :node/after-start (spec/nilable vector?))
 (spec/def :node/after-stop (spec/nilable vector?))
 
+;;;;OFFLINE INBOX
+(spec/def :inbox/wnode (spec/map-of keyword? string?))
```

Comment:
Specs, good stuff

Reply:
What about enode?

Reply:
not sure i'm getting, do you mean we should spec enode address format to conform to pattern like `enode://hash@ip:port`?

Reply:
Just meant a spec in general. I think just saying it is a string is good enough for now, can put comment re regex format as TODO for later